test/compiler-sequencetype-with-uqnames.xspec

```diff
@@ -1,124 +1,134 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<x:description stylesheet="../src/compiler/xquery/declare-variable/sequencetype-with-uqnames.xsl"
+<x:description stylesheet="../src/compiler/base/declare-variable/sequencetype-with-uqnames.xsl"
 	xmlns:x="http://www.jenitennison.com/xslt/xspec">
 
 	<x:helper stylesheet="../src/compiler/base/util/compiler-eqname-utils.xsl"/>
 
-	<x:scenario label="Lexical QName as the entire @as value">
-		<x:scenario label="Declaring namespace on the context element">
-			<x:call function="x:lexical-to-UQName-in-sequence-type">
-				<x:param>
-					<element xmlns:a="urn:x-xspec:example" as="a:b"/>
-				</x:param>
-			</x:call>
-			<x:expect label="enables correct conversion to UQName"
-				select="'Q{urn:x-xspec:example}b'"/>
-		</x:scenario>
-		<x:scenario label="Declaring namespace on ancestor of context element">
-			<x:call function="x:lexical-to-UQName-in-sequence-type">
-				<x:param select="//element">
-					<parent xmlns:a="urn:x-xspec:example">
-						<element as="a:b"/>
-					</parent>
-				</x:param>
-			</x:call>
-			<x:expect label="enables correct conversion to UQName"
-				select="'Q{urn:x-xspec:example}b'"/>
-		</x:scenario>
-		<x:scenario label="When prefix is declared differently on ancestor of context element">
-			<x:call function="x:lexical-to-UQName-in-sequence-type">
-				<x:param select="//element">
-					<parent xmlns:a="urn:x-xspec:example:parent">
+	<x:scenario label="Lexical QName in @as attribute">
+		<x:call function="x:lexical-to-UQName-in-sequence-type">
+			<x:param position="2" select="'as'"/>
```

Comment:
From the diff, it looks like this file had major changes, but really I was adapting the test to the function's 2nd input argument and the file's change in directory location.